.conan/test_package/test_package.cpp

```diff
@@ -0,0 +1,11 @@
+#include <linear_algebra.hpp>
+
+using namespace STD_LA;
+
+int main()
+{
+    vector<fs_vector_engine<double, 4>> vector;
```

Comment:
This is enough to prove that depending on the Conan package and the generated export target include the transitive dependency info needed to use the package.